examples/Interactive/Basic/test_plan.py

```diff
@@ -28,26 +28,7 @@ def main(plan):
 # INTERACTIVE MODE DEMO:
 # ----------------------
 #
-# When HTTP handler starts listening on <IP>:$PORT
-# use a tool like curl to send HTTP requests and execute/reload tests.
-#
-# First execute the tests:
-#     curl -X POST http://127.0.0.1:$PORT/sync/run_tests
-#
-# Make an an edit in my_tests/dependency.py
-#   VALUE = 3
-#     change to:
-#   VALUE = 1
-#
-# Reload the code:
-#     curl -X POST http://127.0.0.1:$PORT/sync/reload
-#
-# Re-run the tests:
-#     curl -X POST http://127.0.0.1:$PORT/sync/run_tests
-#
-# Run only one suite:
-#     curl -X POST http://127.0.0.1:$PORT/sync/run_test_suite -d '{"test_uid": "Test1", "suite_uid": "BasicSuite"}'
-#
-# .. and all other operations that Testplan interactive provides.
-
+# View the API schema at either the localhost or LAN address printed when this
+# testplan script is run. The schema is interactive so can be used to try out
+# the API. To run a test, set its status to "running" on a PUT update.
```

Comment:
Reads better:

You can browse the API schema at either localhost or at the LAN address that's 
printed when running this testplan script. The API schema is interactive so you can test out
what functionality is available. In order to run a test, issue a PUT request
with the test's status set to "running".

Reply:
Thanks for the suggestion, I've made the change.